src/aircraft_cmd.cpp

```diff
@@ -949,7 +949,13 @@ static bool AircraftController(Aircraft *v)
 			return false;
 		}
 
-		/* Vehicle is now at the airport. */
+		/* Vehicle is now at the airport.
+		 * Make sure helicopters actually think they're landing at the tile they're actually on */
```

Comment:
Sentence is missing a dot :P

The comment didn't really make me feel any better (the "think" part, I guess). Do I understand it correctly that it does this:

`Helicopter arrived at his landing pad, so his current position is also where he should land` 

?